File: 217-contains-duplicate/217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> nums_hash = new HashSet<int>();
        for(int i=0;i<nums.Length;i++)
        {
            if(nums_hash.Contains(nums[i]))
            {
                return true;
            }
            else
            {
                nums_hash.Add(nums[i]);
            }
        }
        return false;
    }
}

File: 217-contains-duplicate/217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> hash1 = new HashSet<Integer>();
        for(int i = 0;i < nums.length; i++)
        {
            if(hash1.contains(nums[i]))
               {
                   return true;
               }
              else
               {
                   hash1.add(nums[i]);
               }
        }
        return false;
    }
}